pyproject.toml

```diff
@@ -52,7 +52,7 @@ sanic = {version = ">=20.12.2,<22.0.0", optional = true}
 aiohttp = {version = "^3.7.4.post0", optional = true}
 fastapi = {version = ">=0.65.2", optional = true}
 channels = {version = ">=3.0.5", optional = true}
-"backports.cached-property" = "^1.0.1"
+backports-cached-property = {version = "^1.0.2", python = ">=3.6,<3.8"}
```

Comment:
just curious about the change in the name, was it just cosmetic or there a reason for it?

also I think we can remove the lower bound limit, since we don't support python 3.6 anyway 😊

Reply:
I agree about the lower bound, will remove that.

About the name, I've had a problem with the module not being able to be resolved on my work PC that runs Windows. I'm not sure if the issue was due to the private repo we're using there or Windows. On Linux it resolves fine and even if I do `poetry add backports-cached-property` it's added to pyproject.toml as `backports.cached-property`. However, I think under Windows it's added as `backports-cached-property`. I can check and confirm that tomorrow.

In any case, I should not have a problem at work anymore, since we use Python > 3.8, so I don't really mind either way, although I do think that `backports-cached-property` without the quotes is a bit nicer. Let me know what you prefer and I'll change it accordingly :)

Reply:
I've reverted the name as it was causing a random failure; https://github.com/strawberry-graphql/strawberry/runs/7974876734